### ISSUE TITLE
Use the main element

### ISSUE THREAD

**Issue by LJWatson** [w3c/Unitas#46]
The main content area of each page should be represented by the ```<main>``` element. Currently a ```<div>``` element is used, but this has no semantic meaning from an accessibility point of view.

When the ```<main>``` element is used, screen readers will indicate where the main content area of the page starts/stops. It also provides a hook that screen readers can use as a navigational hook to help people move efficiently to the main content of the page.

**Comment by tripu** [comment]
How could I miss that one?&nbsp;&mdash;&nbsp;I'm a big fan of `main` :) Thanks!
